<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form3</class>
 <widget class="QWidget" name="form3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>495</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(251, 242, 255);</string>
  </property>
  <widget class="QLabel" name="LblMerge">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>48</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>MERGE SORT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowSteps">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(204, 194, 207);
gridline-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>SHOW STEPS</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1Merge">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>36</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 1px solid black;
color: black;
padding: 3px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2Merge">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 1px solid black;
color: black;
padding: 3px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl3Merge">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 1px solid black;
color: black;
padding: 3px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl4Merge">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 1px solid black;
color: black;
padding: 3px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl5Merge">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 1px solid black;
color: black;
padding: 3px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
